### PR TITLE
add hacky script to delete broken jobs

### PR DESCRIPTION
**Pull request by BenTheElder**
I just used this to "fix" jobs stuck trying to pull a bad image. We need a better fix long term though

### CODE REVIEW

**Comment by BenTheElder** [comment]
/area prow
/assign @cjwagner 
FYI @ixdy

File: experiment/delete-busted-jobs.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+TESTINFRA_ROOT=$(git rev-parse --show-toplevel)
+cd $TESTINFRA_ROOT
+
+make -C prow get-build-cluster-credentials
+BUSTED_PODS="$(kubectl get po -n=test-pods | grep -i imagepull | cut -d " " -f1)"
```
Comment (ixdy) [inline experiment/delete-busted-jobs.sh]: there's probably some way to do this with `-l` (selector) and `-o` (select only the pod name)...

Comment (BenTheElder) [reply experiment/delete-busted-jobs.sh]: maybe, this is exactly what I threw together when I realized we had 40+ pods that weren't going away :( I don't think we need `grep -i` either if we just fix the case to match. Though I'm not sure if the selector flag is actually powerful enough.

**Comment by BenTheElder** [comment]
/hold
There's no real need to merge this, but we probably want to keep it around until we fix the underlying issue.

**Comment by krzyzacy** [comment]
probably this logic can go into sinker

**Comment by BenTheElder** [comment]
yeah this should probably go in something like sinker https://github.com/kubernetes/test-infra/issues/5995

**Pull request closed without merging**